src/models/user.model.ts

```diff
@@ -0,0 +1,16 @@
+export default class User {
+
+  public id: string;
+  public login: string;
+  public password: string;
+  public age: number;
+  public isDeleted: boolean;
+
+  constructor( login, password, age, isDeleted, id?) {
```

Comment:
I hope typescript's syntax sugar will help to make it shorter:

 ```typescript
constructor(
public login: string, 
public password: string,
public age: number, 
public isDeleted = false, 
id?: string,
) {
```
However, I'd suggest using an object, as User can contain lots of props